### PR TITLE
feat(design-tokens): dedicated package

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**
Getting started with Design Tokens

**What is the chosen solution to this problem?**
Following #3528, get a fresh new package to extract design tokens from Figma

**Please check if the PR fulfills these requirements**

- [ ] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4343,6 +4480,15 @@
     "@webassemblyjs/helper-fsm" "1.9.0"
     "@xtuc/long" "4.2.2"
 
+"@webassemblyjs/wast-printer@1.8.5":
```
Comment (jmfrancois) [inline yarn.lock:4483]: looks like duplicate (CI is supposed to remove them 🤔

File: packages/design-tokens/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "devDependencies": {
+    "@talend/scripts-config-typescript": "^9.11.1",
+    "css-loader": "^5.0.0",
+    "rimraf": "^3.0.2",
+    "sass": "^1.43.4",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^2.0.0",
+    "ts-loader": "^8.3.0",
+    "typescript": "^4.5.3",
+    "webpack": "4.40.2"
```
Comment (jmfrancois) [inline packages/design-tokens/package.json:40]: please use carret ^ syntax here

File: packages/design-tokens/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@talend/design-tokens",
+  "version": "1.0.0",
```
Comment (jmfrancois) [inline packages/design-tokens/package.json:3]: if you do that you will have a 2.0. You should go down to 0.0.0 and changeset will produce a 1.0.0

File: packages/design-tokens/package.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  "description": "Talend Design tokens",
+  "main": "src/index.ts",
```
Comment (jmfrancois) [inline packages/design-tokens/package.json:5]: should be lib/index.js

File: packages/design-tokens/package.json (hunk 3 above)
Comment (frassinier) [reply packages/design-tokens/package.json:3]: Today I learnt
```suggestion
  "version": "0.0.0",
```

File: packages/design-tokens/webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+	const isDev = argv.mode === 'development';
+	return {
+		mode: isDev ? 'development' : 'production',
+		entry: {
+			TalendDesignTokens: ['./src/index.ts', './src/index.scss'],
+		},
+		output: {
+			filename: '[name].js',
+			path: path.resolve(__dirname, './dist'),
+			libraryTarget: 'umd',
```
Comment (jmfrancois) [inline packages/design-tokens/webpack.config.js:16]:
```suggestion
			library: 'TalendDesignTokens',
			libraryTarget: 'umd',
```

File: packages/design-tokens/webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+module.exports = (env, argv) => {
+	const isDev = argv.mode === 'development';
+	return {
+		mode: isDev ? 'development' : 'production',
+		entry: {
+			TalendDesignTokens: ['./src/index.ts', './src/index.scss'],
+		},
+		output: {
+			filename: '[name].js',
```
Comment (jmfrancois) [inline packages/design-tokens/webpack.config.js:14]: depend on --dev or --prod, quite dangerous, I would tend to add a .min.js in prod and point to it in the package.json main

**Review by jmfrancois — CHANGES_REQUESTED** [review]

File: packages/design-tokens/package.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@talend/design-tokens",
+  "version": "0.0.0",
+  "description": "Talend Design tokens",
+  "main": "lib/index.js",
+  "types": "lib/inddex.d.ts",
```
Comment (frassinier) [inline packages/design-tokens/package.json:6]:
```suggestion
  "types": "lib/index.d.ts",
```

File: packages/design-tokens/talend-scripts.json [hunk 8]
```diff
@@ -0,0 +1,10 @@
+{
+  "preset": "@talend/scripts-preset-react-lib",
+  "webpack": {
+    "debug": true,
```
Comment (frassinier) [inline packages/design-tokens/talend-scripts.json:4]:
```suggestion
```

File: packages/design-tokens/package.json [hunk 9]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@talend/design-tokens",
+  "version": "0.0.0",
+  "description": "Talend Design tokens",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "mainSrc": "src/index.ts",
+  "scripts": {
+    "pre-release": "rimraf ./dist && yarb build:lib && yarn build:umd:dev && yarn build:umd:prod",
```
Comment (frassinier) [inline packages/design-tokens/package.json:9]:
```suggestion
    "pre-release": "rimraf ./dist && yarn build:lib && yarn build:umd:dev && yarn build:umd:prod",
```

File: packages/design-tokens/package.json [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+    "build:umd:dev": "talend-scripts build:lib:umd --dev",
+    "build:umd:prod": "talend-scripts build:lib:umd",
+    "build:lib": "rimraf ./dist && talend-scripts build:ts:lib",
```
Comment (frassinier) [inline packages/design-tokens/package.json:12]:
```suggestion
    "build:lib": "rimraf ./lib && talend-scripts build:ts:lib",
```

**Review by Benrajalu — DISMISSED** [review]

**Review by jmfrancois — APPROVED** [review]

**Review by Benrajalu — APPROVED** [review]

**Pull request merged**